composer.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "candasm/yii1-mobile-detect-component",
+  "version": "1.0.1",
+  "type": "yii-extension",
+  "description": "MobileDetect library for YiiFramework 1.x",
+  "authors": [
+    {
+      "name": "Candasm",
+      "email": "candas@minareci.net"
+    }
+  ],
+  "config": {
+    "vendor-dir": "protected/vendor/"
+  },
+  "autoload": {
+    "psr-4": {
+      "": "src/"
```

Comment:
YiiFramework 1.x series does not support namespace's and composer autoloader so it's not important.